ignite/contrib/handlers/clearml_logger.py

```diff
@@ -178,6 +180,20 @@ def bypass_mode(cls) -> bool:
         """
         return getattr(cls, "_bypass", bool(os.environ.get("CI")))
 
+    def get_clearml_task(self) -> Union[None, Task]:
```

Comment:
```suggestion
    def get_task(self) -> Union[None, Task]:
```

Reply:
I have changed this as suggested in the new commit, but the reason I initially did not use `get_task` is because clearml.Task has a classmethod that has the same name: `Task.get_task()`.